tensorflow/python/lib/io/file_io.i

```diff
@@ -237,6 +237,67 @@ string ReadFromStream(tensorflow::io::BufferedInputStream* stream,
   return result;
 }
 
+string ReadFromStreamUTF8(tensorflow::io::BufferedInputStream* stream,
+                          size_t bytes,
+                          TF_Status* status) {
+  tensorflow::tstring result;
+  size_t total = 0;
+  while (total < bytes) {
+    // Avoid over read:
+    // 1) read at least bytes (utf8 could be 1/2/3/4)
+    // 2) if string ends with partial utf8 (2/3/4), read remaining (< 4) bytes.
+    // 3) repeat until bytes == utf8 chars
+
+    // 1) read at least bytes - total
```

Comment:
Let's move these to a comment in the header of the function, not in the `while` loop.